src/devices/sprut.ts

```diff
@@ -63,36 +63,38 @@ const fzLocal = {
                 return {noise_detected: msg.data['noiseDetected'] === 1};
             }
         },
-    } as Fz.Converter,
+    } satisfies Fz.Converter,
     occupancy_timeout: {
         cluster: 'msOccupancySensing',
         type: ['readResponse', 'attributeReport'],
         convert: (model, msg, publish, options, meta) => {
             return {occupancy_timeout: msg.data['pirOToUDelay']};
         },
-    } as Fz.Converter,
+    } satisfies Fz.Converter,
     noise_timeout: {
         cluster: 'sprutNoise',
         type: ['readResponse', 'attributeReport'],
         convert: (model, msg, publish, options, meta) => {
             return {noise_timeout: msg.data['noiseAfterDetectDelay']};
         },
-    } as Fz.Converter,
+    } satisfies Fz.Converter,
     occupancy_sensitivity: {
         cluster: 'msOccupancySensing',
         type: ['readResponse', 'attributeReport'],
         convert: (model, msg, publish, options, meta) => {
             return {occupancy_sensitivity: msg.data['sprutOccupancySensitivity']};
         },
-    } as Fz.Converter,
+    } satisfies Fz.Converter,
     noise_detect_level: {
         cluster: 'sprutNoise',
         type: ['readResponse', 'attributeReport'],
         convert: (model, msg, publish, options, meta) => {
             return {noise_detect_level: msg.data['noiseDetectLevel']};
         },
-    } as Fz.Converter,
+    } satisfies Fz.Converter,
     co2_mh_z19b_config: {
+        // FIXME: There is no "key" on type Converter
+        // @ts-expect-error
         key: ['co2_autocalibration', 'co2_manual_calibration'],
```

Comment:
`key` is ignored for an Fz.Converter, it can be safely removed

Reply:
Fixed